buildSrc/src/main/java/quilt/internal/tasks/mappings/EnigmaMappingsServerTask.java

```diff
@@ -0,0 +1,62 @@
+package quilt.internal.tasks.mappings;
+
+import org.gradle.api.Project;
+import org.gradle.api.file.RegularFileProperty;
+import org.gradle.api.tasks.InputFile;
+import org.gradle.api.tasks.JavaExec;
+import quilt.internal.Constants;
+import quilt.internal.tasks.MappingsTask;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EnigmaMappingsServerTask extends JavaExec implements MappingsTask {
+	@InputFile
+	private final RegularFileProperty jarToMap;
+
+	public EnigmaMappingsServerTask() {
+		this.setGroup(Constants.Groups.MAPPINGS_GROUP);
+		this.getMainClass().set("org.quiltmc.enigma.network.DedicatedEnigmaServer");
+		this.classpath(getProject().getConfigurations().getByName("enigmaRuntime"));
+		this.jvmArgs("-Xmx2048m");
+
+		this.jarToMap = getObjectFactory().fileProperty();
+	}
+
+	@Override
+	public void exec() {
+		Project project = this.getProject();
```

Comment:
`project` shouldn't be accessed from a task action
I think instead of overriding `exec` this could all just go in a constructor?

Reply:
Fixed it!

Reply:
nvm

Reply:
Now it should actually work